internal/install/types/install_context.go

```diff
@@ -18,7 +21,8 @@ type InstallerContext struct {
 	RecipePaths []string
 	// LocalRecipes is the path to a local recipe directory from which to load recipes.
 	LocalRecipes string
-	Tags         []string
+	// Tags         []string
```

Comment:
Is this meant to be commented out? I would think this either should be included or deleted

Reply:
Good catch, I forgot to remove as I was working on the implementation and test.